test/fixtures/test-runner/output/global_after_should_fail_the_test.snapshot

```diff
@@ -0,0 +1,34 @@
+this is a test
+TAP version 13
+# Subtest: this is a test
+ok 1 - this is a test
+  ---
+  duration_ms: *
+  ...
+not ok NaN - /Users/rluvaton/dev/open-source/node/node-fork/test/fixtures/test-runner/output/global_after_should_fail_the_test.js
```

Comment:
```suggestion
not ok 2 - /Users/rluvaton/dev/open-source/node/node-fork/test/fixtures/test-runner/output/global_after_should_fail_the_test.js
```
my bad - this should not be NaN - we should probably take `this.subtests.length`